### ISSUE TITLE
background when loggin in

### ISSUE THREAD

**Issue by albertocastillo2001** [neutrinolabs/xrdp#1370]
Hello

I have Ubuntu Server 16.04 with lubuntu-desktop installed.

Followed these instructions here https://github.com/neutrinolabs/xrdp/wiki/Building-on-Debian-8
Replaced the file packages with xorgxrdp-0.2.10.tar.gz and xrdp-0.9.10.tar.gz instead

Not sure about this:
Note for Ubuntu 16.04: if your system is using -hwe-16.04 suffixed packages (for example linux-generic-hwe-16.04 and xserver-xorg-core-hwe-16.04), then the -dev packages also need to be -hwe-16.04 suffixed (i.e. make sure xserver-xorg-dev is not installed and xserver-xorg-dev-hwe-16.04 is installed instead).

Well. My system was using "linux-generic-hwe-16.04" and "xserver-xorg-core". SO I installed "xserver-xorg-dev-hwe-16.04", that didn't work, so I installed the non hwe version. Which doesn't work as well.

Since my system is using a mix, what do I do? (In case this is what causing trouble)

Rebooted the server and tried to connect, got the login screen, logged in and I see a blueish background. Nothing else

Saw the log at the user's folder 

`
[   379.909] 
X.Org X Server 1.18.4
Release Date: 2016-07-19
[   379.909] X Protocol Version 11, Revision 0
[   379.909] Build Operating System: Linux 4.4.0-128-generic x86_64 Ubuntu
[   379.909] Current Operating System: Linux vmi280887.contaboserver.net 4.15.0-54-generic #58~16.04.1-Ubuntu SMP Mon Jun 24 13:21:41 UTC 2019 x86_64
[   379.909] Kernel command line: BOOT_IMAGE=/vmlinuz-4.15.0-54-generic root=UUID=768142bd-9f43-40e4-94cc-814dadb11c4f ro nomodeset
[   379.909] Build Date: 10 August 2018  09:33:05AM
[   379.909] xorg-server 2:1.18.4-0ubuntu0.8 (For technical support please see http://www.ubuntu.com/support) 
[   379.909] Current version of pixman: 0.33.6
[   379.910] 	Before reporting problems, check http://wiki.x.org
	to make sure that you have the latest version.
[   379.910] Markers: (--) probed, (**) from config file, (==) default setting,
	(++) from command line, (!!) notice, (II) informational,
	(WW) warning, (EE) error, (NI) not implemented, (??) unknown.
[   379.910] (++) Log file: ".xorgxrdp.10.log", Time: Tue Jul  9 18:35:23 2019
[   379.928] (++) Using config file: "/etc/X11/xrdp/xorg.conf"
[   379.929] (==) Using system config directory "/usr/share/X11/xorg.conf.d"
[   379.929] (==) ServerLayout "X11 Server"
[   379.929] (**) |-->Screen "Screen (xrdpdev)" (0)
[   379.929] (**) |   |-->Monitor "Monitor"
[   379.929] (**) |   |-->Device "Video Card (xrdpdev)"
[   379.929] (**) |-->Input Device "xrdpMouse"
[   379.929] (**) |-->Input Device "xrdpKeyboard"
[   379.929] (**) Option "DontVTSwitch" "on"
[   379.929] (**) Option "AutoAddDevices" "off"
[   379.929] (**) Not automatically adding devices
[   379.929] (==) Automatically enabling devices
[   379.929] (==) Automatically adding GPU devices
[   379.929] (==) Max clients allowed: 256, resource mask: 0x1fffff
[   379.929] (WW) The directory "/usr/share/fonts/X11/cyrillic" does not exist.
[   379.929] 	Entry deleted from font path.
[   379.929] (WW) The directory "/usr/share/fonts/X11/75dpi/" does not exist.
[   379.929] 	Entry deleted from font path.
[   379.929] (WW) The directory "/usr/share/fonts/X11/75dpi" does not exist.
[   379.929] 	Entry deleted from font path.
[   379.929] (==) FontPath set to:
	/usr/share/fonts/X11/misc,
	/usr/share/fonts/X11/100dpi/:unscaled,
	/usr/share/fonts/X11/Type1,
	/usr/share/fonts/X11/100dpi,
	built-ins
[   379.929] (==) ModulePath set to "/usr/lib/x86_64-linux-gnu/xorg/extra-modules,/usr/lib/xorg/extra-modules,/usr/lib/xorg/modules"
[   379.929] (II) Loader magic: 0x55fd18eaddc0
[   379.929] (II) Module ABI versions:
[   379.929] 	X.Org ANSI C Emulation: 0.4
[   379.929] 	X.Org Video Driver: 20.0
[   379.929] 	X.Org XInput driver : 22.1
[   379.929] 	X.Org Server Extension : 9.0
[   379.931] (--) using VT number 2

[   379.931] (II) systemd-logind: logind integration requires -keeptty and -keeptty was not provided, disabling logind integration
[   379.935] (--) PCI:*(0:0:2:0) 1234:1111:1af4:1100 rev 2, Mem @ 0xfd000000/16777216, 0xfea50000/4096, BIOS @ 0x????????/131072
[   379.935] (II) "glx" will be loaded. This was enabled by default and also specified in the config file.
[   379.935] (II) LoadModule: "dbe"
[   379.935] (II) Module "dbe" already built-in
[   379.935] (II) LoadModule: "ddc"
[   379.935] (II) Module "ddc" already built-in
[   379.935] (II) LoadModule: "extmod"
[   379.935] (II) Module "extmod" already built-in
[   379.935] (II) LoadModule: "glx"
[   379.936] (II) Loading /usr/lib/xorg/modules/extensions/libglx.so
[   379.937] (II) Module glx: vendor="X.Org Foundation"
[   379.937] 	compiled for 1.18.4, module version = 1.0.0
[   379.937] 	ABI class: X.Org Server Extension, version 9.0
[   379.937] (==) AIGLX enabled
[   379.937] (II) LoadModule: "int10"
[   379.938] (II) Loading /usr/lib/xorg/modules/libint10.so
[   379.938] (II) Module int10: vendor="X.Org Foundation"
[   379.938] 	compiled for 1.18.4, module version = 1.0.0
[   379.938] 	ABI class: X.Org Video Driver, version 20.0
[   379.938] (II) LoadModule: "record"
[   379.938] (II) Module "record" already built-in
[   379.938] (II) LoadModule: "vbe"
[   379.938] (II) Loading /usr/lib/xorg/modules/libvbe.so
[   379.938] (II) Module vbe: vendor="X.Org Foundation"
[   379.938] 	compiled for 1.18.4, module version = 1.1.0
[   379.938] 	ABI class: X.Org Video Driver, version 20.0
[   379.938] (II) LoadModule: "xorgxrdp"
[   379.938] (II) Loading /usr/lib/xorg/modules/libxorgxrdp.so
[   379.939] (II) Module XORGXRDP: vendor="X.Org Foundation"
[   379.939] 	compiled for 1.19.6, module version = 0.2.10
[   379.939] 	ABI class: X.Org Video Driver, version 23.0
[   379.939] (EE) module ABI major version (23) doesn't match the server's version (20)
[   379.939] (II) UnloadModule: "xorgxrdp"
[   379.939] (II) Unloading xorgxrdp
[   379.939] (EE) Failed to load module "xorgxrdp" (module requirement mismatch, 0)
[   379.939] (II) LoadModule: "fb"
[   379.939] (II) Loading /usr/lib/xorg/modules/libfb.so
[   379.939] (II) Module fb: vendor="X.Org Foundation"
[   379.939] 	compiled for 1.18.4, module version = 1.0.0
[   379.939] 	ABI class: X.Org ANSI C Emulation, version 0.4
[   379.939] (II) LoadModule: "xrdpdev"
[   379.939] (II) Loading /usr/lib/xorg/modules/drivers/xrdpdev_drv.so
[   379.939] (EE) Failed to load /usr/lib/xorg/modules/drivers/xrdpdev_drv.so: /usr/lib/xorg/modules/drivers/xrdpdev_drv.so: undefined symbol: rdpRRModeDestroy
[   379.939] (II) UnloadModule: "xrdpdev"
[   379.939] (II) Unloading xrdpdev
[   379.939] (EE) Failed to load module "xrdpdev" (loader failed, 7)
[   379.939] (==) Matched modesetting as autoconfigured driver 0
[   379.939] (==) Matched fbdev as autoconfigured driver 1
[   379.939] (==) Matched vesa as autoconfigured driver 2
[   379.939] (==) Assigned the driver to the xf86ConfigLayout
[   379.939] (II) LoadModule: "modesetting"
[   379.939] (II) Loading /usr/lib/xorg/modules/drivers/modesetting_drv.so
[   379.940] (II) Module modesetting: vendor="X.Org Foundation"
[   379.940] 	compiled for 1.18.4, module version = 1.18.4
[   379.940] 	Module class: X.Org Video Driver
[   379.940] 	ABI class: X.Org Video Driver, version 20.0
[   379.940] (II) LoadModule: "fbdev"
[   379.940] (II) Loading /usr/lib/xorg/modules/drivers/fbdev_drv.so
[   379.940] (II) Module fbdev: vendor="X.Org Foundation"
[   379.940] 	compiled for 1.18.1, module version = 0.4.4
[   379.940] 	Module class: X.Org Video Driver
[   379.940] 	ABI class: X.Org Video Driver, version 20.0
[   379.940] (II) LoadModule: "vesa"
[   379.940] (II) Loading /usr/lib/xorg/modules/drivers/vesa_drv.so
[   379.940] (II) Module vesa: vendor="X.Org Foundation"
[   379.940] 	compiled for 1.18.1, module version = 2.3.4
[   379.940] 	Module class: X.Org Video Driver
[   379.940] 	ABI class: X.Org Video Driver, version 20.0
[   379.940] (II) LoadModule: "xrdpmouse"
[   379.940] (II) Loading /usr/lib/xorg/modules/input/xrdpmouse_drv.so
[   379.940] (II) Module XRDPMOUSE: vendor="X.Org Foundation"
[   379.940] 	compiled for 1.19.6, module version = 0.2.10
[   379.940] 	Module class: X.Org XInput Driver
[   379.940] 	ABI class: X.Org XInput driver, version 24.1
[   379.940] (EE) module ABI major version (24) doesn't match the server's version (22)
[   379.940] (II) UnloadModule: "xrdpmouse"
[   379.940] (II) Unloading xrdpmouse
[   379.940] (EE) Failed to load module "xrdpmouse" (module requirement mismatch, 0)
[   379.940] (II) LoadModule: "xrdpkeyb"
[   379.941] (II) Loading /usr/lib/xorg/modules/input/xrdpkeyb_drv.so
[   379.941] (II) Module XRDPKEYB: vendor="X.Org Foundation"
[   379.941] 	compiled for 1.19.6, module version = 0.2.10
[   379.941] 	Module class: X.Org XInput Driver
[   379.941] 	ABI class: X.Org XInput driver, version 24.1
[   379.941] (EE) module ABI major version (24) doesn't match the server's version (22)
[   379.941] (II) UnloadModule: "xrdpkeyb"
[   379.941] (II) Unloading xrdpkeyb
[   379.941] (EE) Failed to load module "xrdpkeyb" (module requirement mismatch, 0)
[   379.941] (II) modesetting: Driver for Modesetting Kernel Drivers: kms
[   379.941] (II) FBDEV: driver for framebuffer: fbdev
[   379.941] (II) VESA: driver for VESA chipsets: vesa
[   380.064] (EE) open /dev/dri/card0: No such file or directory
[   380.064] (WW) Falling back to old probe method for modesetting
[   380.064] (EE) open /dev/dri/card0: No such file or directory
[   380.064] (II) Loading sub module "fbdevhw"
[   380.064] (II) LoadModule: "fbdevhw"
[   380.064] (II) Loading /usr/lib/xorg/modules/libfbdevhw.so
[   380.065] (II) Module fbdevhw: vendor="X.Org Foundation"
[   380.065] 	compiled for 1.18.4, module version = 0.0.2
[   380.065] 	ABI class: X.Org Video Driver, version 20.0
[   380.065] (**) FBDEV(1): claimed PCI slot 0@0:2:0
[   380.065] (II) FBDEV(1): using default device
[   380.065] (WW) Falling back to old probe method for vesa
[   380.065] (EE) Screen 0 deleted because of no matching config section.
[   380.065] (II) UnloadModule: "modesetting"
[   380.065] (**) FBDEV(0): Depth 24, (--) framebuffer bpp 32
[   380.065] (==) FBDEV(0): RGB weight 888
[   380.065] (==) FBDEV(0): Default visual is TrueColor
[   380.065] (==) FBDEV(0): Using gamma correction (1.0, 1.0, 1.0)
[   380.065] (II) FBDEV(0): hardware: EFI VGA (video memory: 1408kB)
[   380.065] (II) FBDEV(0): checking modes against framebuffer device...
[   380.065] (II) FBDEV(0): 	mode "640x480" test failed
[   380.065] (II) FBDEV(0): 	mode "640x480" test failed
[   380.065] (II) FBDEV(0): 	mode "640x480" test failed
[   380.065] (II) FBDEV(0): 	mode "640x480" test failed
[   380.065] (II) FBDEV(0): 	mode "640x480" test failed
[   380.065] (II) FBDEV(0): 	mode "640x480" test failed
[   380.065] (II) FBDEV(0): 	mode "640x480" not found
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" test failed
[   380.065] (II) FBDEV(0): 	mode "800x600" not found
[   380.065] (II) FBDEV(0): 	mode "1024x768" test failed
[   380.065] (II) FBDEV(0): 	mode "1024x768" test failed
[   380.065] (II) FBDEV(0): 	mode "1024x768" test failed
[   380.065] (II) FBDEV(0): 	mode "1024x768" test failed
[   380.065] (II) FBDEV(0): 	mode "1024x768" test failed
[   380.065] (II) FBDEV(0): 	mode "1024x768" test failed
[   380.065] (II) FBDEV(0): 	mode "1024x768" test failed
[   380.065] (II) FBDEV(0): 	mode "1024x768" not found
[   380.065] (II) FBDEV(0): 	mode "1280x720" test failed
[   380.065] (II) FBDEV(0): 	mode "1280x720" not found
[   380.065] (II) FBDEV(0): 	mode "1280x1024" test failed
[   380.065] (II) FBDEV(0): 	mode "1280x1024" test failed
[   380.065] (II) FBDEV(0): 	mode "1280x1024" test failed
[   380.065] (II) FBDEV(0): 	mode "1280x1024" not found
[   380.065] (II) FBDEV(0): 	mode "1600x900" test failed
[   380.065] (II) FBDEV(0): 	mode "1600x900" not found
[   380.065] (II) FBDEV(0): 	mode "1920x1080" test failed
[   380.065] (II) FBDEV(0): 	mode "1920x1080" not found
[   380.065] (II) FBDEV(0): checking modes against monitor...
[   380.065] (--) FBDEV(0): Virtual size is 800x600 (pitch 800)
[   380.065] (**) FBDEV(0):  Built-in mode "current": 48.0 MHz, 46.9 kHz, 75.1 Hz
[   380.065] (II) FBDEV(0): Modeline "current"x0.0   48.00  800 832 928 1024  600 604 608 624 -hsync -vsync -csync (46.9 kHz b)
[   380.065] (==) FBDEV(0): DPI set to (96, 96)
[   380.065] (II) Loading sub module "fb"
[   380.065] (II) LoadModule: "fb"
[   380.066] (II) Loading /usr/lib/xorg/modules/libfb.so
[   380.066] (II) Module fb: vendor="X.Org Foundation"
[   380.066] 	compiled for 1.18.4, module version = 1.0.0
[   380.066] 	ABI class: X.Org ANSI C Emulation, version 0.4
[   380.066] (**) FBDEV(0): using shadow framebuffer
[   380.066] (II) Loading sub module "shadow"
[   380.066] (II) LoadModule: "shadow"
[   380.066] (II) Loading /usr/lib/xorg/modules/libshadow.so
[   380.066] (II) Module shadow: vendor="X.Org Foundation"
[   380.066] 	compiled for 1.18.4, module version = 1.1.0
[   380.066] 	ABI class: X.Org ANSI C Emulation, version 0.4
[   380.066] (II) UnloadModule: "vesa"
[   380.066] (II) Unloading vesa
[   380.066] (==) Depth 24 pixmap format is 32 bpp
[   380.066] (EE) FBDEV(0): FBIOPUT_VSCREENINFO succeeded but modified mode
[   380.066] (EE) FBDEV(0): mode initialization failed
[   380.066] (EE) 
Fatal server error:
[   380.066] (EE) AddScreen/ScreenInit failed for driver 0
[   380.066] (EE) 
[   380.066] (EE) 
Please consult the The X.Org Foundation support 
	 at http://wiki.x.org
 for help. 
[   380.066] (EE) Please also check the log file at ".xorgxrdp.10.log" for additional information.
[   380.067] (EE) 
[   380.069] (EE) Server terminated with error (1). Closing log file.
`

What's going on?

Thank you.

**Comment by metalefty** [comment]
This is the cause:
```
[ 379.941] (EE) module ABI major version (24) doesn't match the server's version (22)
```

The solution is written in the wiki page. You just ignored that.

**Comment by metalefty** [comment]
Likely to be same as neutrinolabs/xorgxrdp#100

**Comment by albertocastillo2001** [comment]
@metalefty thanks for your response.

However I hav a question about that which I posted above.
Ran a dpkg-query --search hwe command and found out that these are the hwe packages I have installed by default:

> linux-image-generic-hwe-16.04: /usr/share/doc/linux-image-generic-hwe-16.04/copyright
linux-generic-hwe-16.04: /usr/share/doc/linux-generic-hwe-16.04/changelog.gz
linux-headers-generic-hwe-16.04: /usr/share/doc/linux-headers-generic-hwe-16.04/copyright
linux-generic-hwe-16.04: /usr/share/doc/linux-generic-hwe-16.04/copyright
linux-headers-generic-hwe-16.04: /usr/share/doc/linux-headers-generic-hwe-16.04
linux-image-generic-hwe-16.04: /usr/share/doc/linux-image-generic-hwe-16.04/changelog.gz
linux-image-generic-hwe-16.04: /usr/share/doc/linux-image-generic-hwe-16.04
linux-generic-hwe-16.04: /usr/share/doc/linux-generic-hwe-16.04
linux-headers-generic-hwe-16.04: /usr/share/doc/linux-headers-generic-hwe-16.04/changelog.gz

As you see, xserver-xorg-core is not hwe-16.04 suffixed.  I initially installed xserver-xorg-dev-hwe-16.04, but that didn't work. I now replaced the package by the non suffixed version but that doesn't work either. I have a mix of packages installed by default, 

So I tried both options available and that didn't work
